engine/resource/pipeline.go

```diff
@@ -36,6 +36,7 @@ type Pipeline struct {
 	Platform    manifest.Platform    `json:"platform,omitempty"`
 	Trigger     manifest.Conditions  `json:"conditions,omitempty"`
 
+	Resources   StageResources    `json:"resources,omitempty"`
```

Comment:
instead of a new type, we could just re-use the `Resources` struct and ignore limits

Reply:
Fixed